Quick_Deploy/vLLM/model_repository/vllm/config.pbtxt

```diff
@@ -41,17 +41,18 @@ model_transaction_policy {
 
 input [
   {
-    name: "PROMPT"
+    name: "prompt"
```

Comment:
@nnshah1 after our offline discussion, I'm under the impression you want this to be called `text` instead of `prompt` for both input and output.

I think this works fine for the model standalone - however for defining ensembles, I think having an input and an output with the same name may cause issues with mapping step I/O

Reply:
Yes. We should avoid same name for two different tensors. All tensors should have unique identifiable names.